src/directives/OcTooltip.js

```diff
@@ -49,9 +61,11 @@ export default {
       ignoreAttributes: true,
       ...(Object.prototype.toString.call(value) === "[object Object]" && value),
       aria: {
-        content: "describedby",
+        content: null,
+        expanded: false,
+        hidden: true,
```

Comment:
this can go because tippy doesn't support the option and we manage it our own by ariaHidden plugin

Reply:
Thanks, should be resolved!